src/locales/ko.js

```diff
@@ -0,0 +1,56 @@
+export default {
+    title: '유니온 지도 메이커',
+    instructions: '설명:',
+    instructionsSub1: '1. 유니온 지도를 더 빨리 채우려면 지역 선택을 활성화해 줍니다.',
+    instructionsSub2: '2. 유니온 지도에 채우려는 직업별 모양의 양을 입력해 줍니다.',
+    instructionsSub3: '3. 산정된 공간은 선택된 공간의 양과 같아야 하며 같지 않다면 프로그램은 계속 반복 실행할 겁니다.',
+    instructionsSub4: '4. 프로그램이 루트를 찾고 있을 때 해당 과정을 보고 싶다면 실시간 보기를 활성화해 줍니다.',
+
+    spacesToBeFilled: '유니온 캐릭터 산정된 공간: ',
+    boardSpacesFilled: '유니온 지도 선택된 공간: ',
+    iterations: '반복횟수: ',
+    time: '소요시간: ',
+
+    bigClick: '지역선택',
+    liveSolve: '실시간 보기',
+    darkMode: '다크모드',
+
+    start: '시작',
+    pause: 'Pause',
+    continue: 'Continue',
```

Comment:
Could you also translate these?

Reply:
I wrote down the English word that I often use in Korea. Should I apply it if you need it?